app/controllers/miq_request_controller.rb

```diff
@@ -415,16 +415,26 @@ def get_request_tab_type
     end
   end
 
+  def remote_and_global_requestors(requestor)
```

Comment:
the method works fine but it is mixing couple things which can separate.
it is mixing query and building condition and the name is not reflections what is done in the method. (it is returning part of condition)

so I would do:

1.Create scope method or the instance method on User model to get all user according `userid` independently on the region.
something like 
```
  def self.find_remote_and_global_by(params)
    ..
  end
or
  def self.with_remote_and_global_users_by(params)
    where(params)
    ..
  end
or 
...
```
Not sure about naming

2. building condition
you are solving the case if there is an array of requester ids or if not.
I can method imagine method on Condition class like
```ruby
def self.array_condtions(ids, field)
   if ids.count == 1
     {"or" => ids.collect { |id| {"=" => {"value" => id, "field" => field}} }
  esle
     { "=" => {"value" => ids.first, "field" => field}
  end
end
```
3. and use these methods in controller
`
  cond.push( Condition.array_condtions(User.with_remote_and_global_users_by({:userid => ...}, "MiqRequest-requester_id"}))
...
`
Implementaion of methods is up to you I just want suggest struture. 

but it will lead to more PRs maybe it be done in follow up PR.